scripts/bench/server.js

```diff
@@ -21,7 +21,7 @@ function createHTTP2Server(benchmark) {
       __dirname,
       'benchmarks',
       benchmark,
-      request.url
+      request.url,
```

Comment:
This won't work. Everything inside `scripts/bench` (and `scripts` in general, except `scripts/rollup/shims`) should be normal ES6 without trailing commas in function calls. Otherwise it won't run on Node.